app/apollo/resolvers/clusterDistributed.js

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright 2020 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const lodash = require('lodash');
+const Moment = require('moment');
+const { AuthenticationError } = require('apollo-server');
+const { ACTIONS, TYPES } = require('../models/const');
+
+const buildSearchForClusterName = (ordId, searchStr) => {
+  const tokens = searchStr.split(/\s+/);
+  const ands = lodash.map(tokens, token => {
```

Comment:
I think `tokens.map()` will work for you without adding the dependency on lodash

Reply:
fixed, removed logdash dependency in this file.